src/data/zsqrtd/basic.lean

```diff
@@ -5,6 +5,7 @@ Authors: Mario Carneiro
 -/
 import algebra.associated
 import tactic.ring
+import data.real.sqrt
```

Comment:
This vastly grows the dependencies of this file. Please put this in another file.

Reply:
Fair point, moved.